src/com/amplitude/api/AmplitudeClient.java

```diff
@@ -1025,7 +1034,7 @@ protected long saveEvent(String eventType, JSONObject event) {
             return -1;
         }
 
-        if (eventType.equals(Constants.IDENTIFY_EVENT)) {
+        if (eventType.equals(Constants.IDENTIFY_EVENT) || eventType.equals(Constants.GROUP_IDENTIFY_EVENT)) {
```

Comment:
I assume we diligently tracked down everywhere `IDENTIFY_EVENT` was used.

Reply:
yep